bedrock/mozorg/templates/mozorg/robots.txt

```diff
@@ -6,6 +6,6 @@ disallow: /*/firstrun/
 disallow: /*/newsletter/existing/
 disallow: /*/whatsnew/
 disallow: /*/etc/
-disallow: /*/external/
+disallow: /*/pocket/
```

Comment:
Not sure I understand this. In mozorg mode is this a valid URL? Are we keeping the pocket pages in mozorg mode?

Reply:
I don't think this is a thing in mozorg mode, so I don't think we need this change.

Reply:
I think we're getting rid of all of the existing `/external` URLs, so we can just remove it.

Reply:
Good spot - killing it now

Reply:
Makes me think we need to add a robots.txt for Pocket mode, too